README.md

```diff
@@ -1,18 +1,22 @@
 
 # Stackdriver Sandbox (Alpha)
 
-Stackdriver Sandbox is an open-source tool that helps practitioners to learn Service Reliability Engineering practices from Google and apply them on their cloud services using Stackdriver. It is based on [Hipster Shop](https://github.com/GoogleCloudPlatform/microservices-demo), a cloud-native microservices application.
+Stackdriver Sandbox is an open-source tool that helps practitioners to learn Service Reliability Engineering practices from Google and apply them on their cloud services using [Operations](https://cloud.google.com/products/operations) (formerly Stackdriver).
+It is based on [Hipster Shop](https://github.com/GoogleCloudPlatform/microservices-demo), a cloud-native microservices application.
```

Comment:
Hipster Shop was also recently rebranded to "Online Boutique".

Reply:
Would this be a bigger change for us, maybe for a separate PR? Since the webpage created from our project + all the other docs say Hipster Shop